NOTICE.txt

```diff
@@ -630,9 +630,9 @@ THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR I
 
 --------------------------------------------------------------------
 Dependency: github.com/go-sql-driver/mysql
-Revision: 9dee4ca50b83acdf57a35fb9e6fb4be640afa2f3
+Revision: cd4cb909ce1a31435164be29bf3682031f61539a
```

Comment:
Could you open a separate PR to update the mysql dependency? I think we should do that anyway.